readme.md

```diff
@@ -52,6 +52,15 @@ const execa = require('execa');
 	const {stdout} = await execa.shell('echo unicorns');
 	//=> 'unicorns'
 
+	// Cancelling a spawned process
+	const spawned = execa("node");
```

Comment:
Single-quotes

Reply:
I think `process` would be a better name than `spawned`.

Reply:
Would this confuse readers about the global variable named `process`? Child processes returned by `spawn()` and global `process` have different methods/properties (although some are shared).

Reply:
Ah yeah, `subprocess` is better.